x-pack/plugins/security_solution/server/search_strategy/timeline/factory/events/all/constants.ts

```diff
@@ -239,5 +232,8 @@ export const TIMELINE_EVENTS_FIELDS = [
   'zeek.ssl.established',
   'zeek.ssl.resumed',
   'zeek.ssl.version',
-  ...TIMELINE_CTI_FIELDS,
+  // TODO in the future, our alerts timeline fields should be derived from the
```

Comment:
is there a ticket for this work?

Reply:
cc @andrew-goldstein

Reply:
> is there a ticket for this work?

API changes for RAC: https://github.com/elastic/kibana/issues/94520